pkg/console/controllers/healthcheck/controller.go

```diff
@@ -149,8 +154,14 @@ func (c *HealthCheckController) Sync(ctx context.Context, controllerContext fact
 
 func (c *HealthCheckController) CheckRouteHealth(ctx context.Context, operatorConfig *operatorsv1.Console, route *routev1.Route) (string, error) {
 	var reason string
+	healthCheckBackoff := wait.Backoff{
+		Steps:    5,
+		Duration: 5 * time.Second,
+		Factor:   1.0,
+		Jitter:   0.1,
+	}
```

Comment:
Not certain but I think this is approximately 25 seconds as written, 5 steps delayed by 5s each time plus some jitter/backoff.

Maybe we should start with 10 seconds to see if we can get this gone, so steps 10, duration 1 second?